config/checkstyle-exclude.xml

```diff
@@ -21,6 +21,9 @@
         "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
+
+    <suppress checks="VisibilityModifier" files="com[\\/]mongodb[\\/]internal[\\/]connection[\\/]tlschannel[\\/]"/>
```

Comment:
Lots of publish methods/fields in the vendored code.

Reply:
Ack